blackjax/sgmcmc.py

```diff
@@ -0,0 +1,17 @@
+from jax import random
+
+from blackjax.inference.sgmcmc.diffusions import overdamped_langevin
+from blackjax.inference.sgmcmc.gradient_estimation import build_gradient_estimation_fn, build_grad_log_post
+
+def build_SGLD_kernel(key, step_size, loglikelihood, logprior, params_IC, data, batch_size):
+    grad_log_post = build_grad_log_post(loglikelihood, logprior, data)
+    estimate_gradient, init_gradient = build_gradient_estimation_fn(grad_log_post, data, batch_size)
+    mygrad, gradient_state = init_gradient(key, params_IC)
+    state = (params_IC, mygrad, gradient_state)
+
+    one_step = overdamped_langevin(estimate_gradient)
+
+    def kernel(rng_key, state):
```

Comment:
When I am working with SgMCMC I expect to be able to pass data iteratively instead of the whole dataset, and thus the following kernel signature:

```python
def kernel(rng_key, state, batch)
```

where `batch` contains the new data points.